emt4j-agent/emt4j-agent-jdk17/src/main/java/org/eclipse/emt4j/agent/jdk17/MainAgent.java

```diff
@@ -0,0 +1,118 @@
+/********************************************************************************
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+package org.eclipse.emt4j.agent.jdk17;
+
+import org.eclipse.emt4j.agent.common.JdkDependConfig;
+import org.eclipse.emt4j.agent.common.methodvisitor.TransformerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.lang.instrument.Instrumentation;
+import java.lang.instrument.UnmodifiableClassException;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.jar.JarFile;
+
+import static org.eclipse.emt4j.agent.common.Constant.*;
+import static org.eclipse.emt4j.agent.common.Constant.COMMON_PACKAGE;
```

Comment:
This line seems redundant since there is a * import already.

Reply:
removed this line